transformers/storage/repository_test.go

```diff
@@ -691,7 +696,7 @@ func insertFlopKick(blockNumber int64, bidId string, contractAddressId int64, db
 	// inserting a flop kick log event record
 	headerId := insertHeader(db, blockNumber)
 	flopKickLog := test_data.CreateTestLog(headerId, db)
-	_, insertErr := db.Exec(flop_kick.InsertFlopKickQuery, headerId, bidId, 0, 0, "", contractAddressId, flopKickLog.ID)
+	_, insertErr := db.Exec(insertFlopKickQuery, headerId, bidId, 0, 0, "", contractAddressId, flopKickLog.ID)
```

Comment:
Not a huge fan of these zero-value arguments without knowing what they are. Assign them to some local variables?